<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWidget</class>
 <widget class="QWidget" name="DashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#infoCard {
    background-color: rgb(255, 255, 255);
    border: 1px solid #e0e0e0;
    border-radius: 8px;
}
QLabel#valueLabel {
    font-size: 28pt;
    font-weight: bold;
    color: #005a9e;
}
QLabel#titleLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #333;
}
QLabel#iconLabel {
    font-size: 32pt;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="infoCard">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>üì¶</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>T·ªïng S·ªë S·∫£n Ph·∫©m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="productsCountLabel">
          <property name="objectName">
           <string>valueLabel</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="infoCard">
     <property name="styleSheet">
      <string notr="true">QLabel#valueLabel {
    color: #e67e22;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>‚ö†Ô∏è</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>S·∫£n Ph·∫©m S·∫Øp H·∫øt H√†ng</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lowStockCountLabel">
          <property name="objectName">
           <string>valueLabel</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="infoCard">
     <property name="styleSheet">
      <string notr="true">QLabel#valueLabel {
    color: #c0392b;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>‚è≥</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>S·∫£n Ph·∫©m S·∫Øp H·∫øt H·∫°n</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="expiringCountLabel">
          <property name="objectName">
           <string>valueLabel</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="infoCard">
     <property name="styleSheet">
      <string notr="true">QLabel#valueLabel {
    color: #27ae60;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>üí∞</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Doanh Thu Th√°ng N√†y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="revenueLabel">
          <property name="objectName">
           <string>valueLabel</string>
          </property>
          <property name="text">
           <string>0 VND</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>